server/src/models/event.ts

```diff
@@ -16,10 +17,13 @@ export class Event {
   @PrimaryGeneratedColumn() id: number;
 
   @CreateDateColumn()
-  createdDate: number;
+  createdDate: string;
```

Comment:
consider checking the proper type: based on the docs it should be `Date`

Reply:
I have fixed all the comments and am very grateful for the review